app/components/candidate_interface/course_choices_review_component.rb

```diff
@@ -246,13 +246,23 @@ def visa_details_row(application_choice)
 
     def immigration_right_to_work?(application_choice)
       application_choice.application_form.british_or_irish? ||
-        application_choice.application_form.immigration_right_to_work?
+        application_form.right_to_work_or_study == 'yes'
```

Comment:
Use the [enum](https://api.rubyonrails.org/v5.1/classes/ActiveRecord/Enum.html) magic magic method here?
```suggestion
        application_form.right_to_work_or_study_yes?
```

Reply:
🤯 🤯 🤯